contracts/TokenIOStorage.sol

```diff
@@ -646,7 +646,7 @@ contract TokenIOStorage is Ownable {
      * @param _fxUSDBPSRate  usdbps rate
      * @return { "uint" : "Returns the uint value associated with the key" }
      */
-    function setTokenfxUSDBPSRate(address _address, uint _fxUSDBPSRate) external returns(bool success) {
+    function setTokenfxUSDBPSRate(address _address, uint _fxUSDBPSRate) external onlyOwner returns(bool success) {
```

Comment:
ok, who is the owner of the storage contract? And where other such permission modifiers could be missing?